app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -148,28 +148,77 @@ private double getPokemonLevelFromImg(Bitmap pokemonImage, int trainerLevel) {
      * If there was no detected upgrade cost, returns -1
      *
      * @param pokemonImage The image of the full pokemon screen
-     * @return the evolution cost, or -1 on scan failure
+     * @return the evolution cost, or -1 on no upgrade cost, or -999 on scan failure
      */
     public int getPokemonEvolutionCostFromImg(Bitmap pokemonImage) {
+        int error = -999;
```

Comment:
Does -999 have a special meaning elsewhere outside this function? I'm trying to see if this can be cleaned up or if we need to define a "final" to denote errors. But returning errors isn't really good Java style. That's what exceptions are for.